xrootd/authorization.go

```diff
@@ -529,6 +540,17 @@ func makeSciTokensCfg() (cfg ScitokensCfg, err error) {
 	return cfg, nil
 }
 
+func resolveAuthorization(auth1 string, auth2 string) string {
+	if auth1 == auth2 {
+		return auth1
+	} else {
+		// Either one is read and the other is write, in which case we want the default of
+		// all, or one is all and the other is read/write, and we'd still want it to be the default
+		// of all, which is equivalent to the empty string
+		return ""
```

Comment:
I think I'm more in favor of using "all" in this case instead of relying on the empty string's default behavior. That should make it a bit easier for Fabio to understand the configs we generate without having to find comments in source code.